config/initializers/time_formats.rb

```diff
@@ -0,0 +1,4 @@
+Time::DATE_FORMATS[:date] = "%-d %B %Y"
+Time::DATE_FORMATS[:time] = "%-l:%M%P"
```

Comment:
You could call this "time_12_hour_clock" ?

Reply:
I wondered about time_with_period but that level of detail for time wouldn't be on other helpers.

Reply:
I think the `%P` means that it always shows "am" or "pm" right? So the name would be consistent for this format. I'm old, so I still call it "12 hour clock" 😄

Reply:
Yup yup - that's officially called a period though no-one uses that term in colloquial language. 12 hour clock is a classic phrase 😁